### PR TITLE
feat: template can use parens to removeHTMLFromString

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

## Testing:
Tested a few template strings directly

### CODE REVIEW

**Comment by jbrundage** [comment]
image

**Comment by GordonSmith** [comment]
I think those regular brackets would be too common to use in this scenario?

**Comment by GordonSmith** [comment]
Is the same true for `{}`?

So does that mean:  `Some str ((a))` => `Some str (123)`

Did we ask if there was ever a situation where they wanted to retain the HTML formatting?

File: src/common/Utility.js [hunk 1]
```diff
@@ -218,50 +218,63 @@
...
+        div.innerHTML = str;
+        return div.textContent || div.innerText || "";
+    }
 
-        if (!args || !args.hasOwnProperty) {
-            args = {};
+//  Template   ---
+//  https://github.com/Matt-Esch/string-template (MIT)
+var nargs = /\{([0-9a-zA-Z_\s\[\]]+)\}/g;
+var nargs2 = /\(([0-9a-zA-Z_\s\[\]]+)\)/g;
```
Comment (GordonSmith) [inline src/common/Utility.js]: How about:  `/\{\{([0-9a-zA-Z_\s\[\]]+)\}\}/g;`
So if {a} then its verbatim, but if its:  {{a}} then it strips the HTML?

Comment (jbrundage) [reply src/common/Utility.js]: That sounds good to me. I'll test it out

**Comment by jbrundage** [comment]
@GordonSmith I've amended the PR to add the `{{var}}` concept. It seems to be working as expected.

**Pull request merged**